lightstep/resource_metric_dashboard.go

```diff
@@ -599,6 +611,78 @@ func buildCharts(chartsIn []interface{}) ([]client.UnifiedChart, error) {
 	return newCharts, nil
 }
 
+var colors = map[string]bool{
+	"#AA3018": true,
+	"#B67D0C": true,
+	"#3C864F": true,
+	"#1B7BBB": true,
+	"#DC7847": true,
+	"#78469B": true,
+	"#37A2AE": true,
+	"#B03B7F": true,
+	"#1F40C1": true,
+	"#8B7255": true,
+	"#826CEF": true,
+	"#D56DD5": true,
+	"#6699CC": true,
+}
+
+var operators = map[string]bool{
+	"E":  true,
+	"GE": true,
+	"GT": true,
+	"LE": true,
+	"LT": true,
+}
```

Comment:
where do consts usually live?

Reply:
i think the `ValidateFunc` above makes this const unnecessary, right?

Reply:
Could/should the colors live in backend validation instead of here? If we add more colors/operators, TF would need to be updated as well.

Reply:
I can remove the validation here and move it to the other PR I have open where I'm doing the updates BE updates for updating

Reply:
i think _if_ we plan on validating colors, it should be here -- that way we can catch invalid colors at plan time instead of waiting until you try to apply. that said, i still think restricting the list of colors is a much worse UX for little benefit